bee-node/docker/bee.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3.9"
+services:
+  bee:
+    build:
+      context: ../../
+      dockerfile: ./bee-node/docker/Dockerfile
+    image: bee:${IMAGE_TAG}
+    working_dir: /bee_workdir
+    tty: true # for colored logs
+    container_name: "bee"
+    ports:
+      - "14265:14265"
+      - "15600:15600"
+      - "1883:1883"
+      - "8081:8081"
```

Comment:
These ports may change depending on content of the user's `config.toml`.

Reply:
I could add some ENV variables inside the `build_and_run` script and pass them to docker-compose file?

Reply:
in `docker/bee.yaml` the ports are now like this
```yaml
ports:
      - ${BINDING_PORT}:${BINDING_PORT}
      - ${BIND_ADDRESS_PORT}:${BIND_ADDRESS_PORT}
      - ${MQTT_PORT}:${MQTT_PORT}
      - ${DASHBOARD_PORT}:${DASHBOARD_PORT}
```
and `build_and_run` has this new lines
```sh
# this are extracted from config.toml and passed to docker/bee.yaml file
export DASHBOARD_PORT=`cat ./config.toml | grep "port" | tail -1 | sed 's/["port=]//g' | tr -d '[[:space:]]'`
export MQTT_PORT=`cat ./config.toml | grep "address"| tail -1 | sed 's/["address=tcp://loclhot:]//g' | tr -d '[[:space:]]'`
export BINDING_PORT=`cat ./config.toml | grep "binding_port" | tail -1 | sed 's/["binding_port=]//g' | tr -d '[[:space:]]'`
export BIND_ADDRESS_PORT=`cat ./config.toml | grep "bind_address"| head -1 | sed 's#.*/\([^:]*\).*#\1#' | sed 's/["]//g' | tr -d '[[:space:]]'`
```
@Adam-Gleave  Let me know if this could be a solution